documentation/modules/exploit/multi/http/magento_unserialize.md

```diff
@@ -68,7 +68,7 @@ If at some point the IP (base URL) of Magento has changed, then you will need to
 
 1. From the terminal, do: ```mysql -h localhost -u [username] -p[password]```
 2. In the SQL prompt, do: ```use [magento database name]```
-3. Do: ```select * from core_config_data;```, you should see both web/unsecure/base_url (config ID 2) and web/secure/base_url (config ID 3) with the hardcoded IP.
+3. Do: ```select * from core_config_data;```, you should see both web/insecure/base_url (config ID 2) and web/secure/base_url (config ID 3) with the hardcoded IP.
```

Comment:
I think this needs reverted https://magento.stackexchange.com/questions/227157/why-is-web-unsecure-base-url-missing-in-core-config-data